db/models/endpointsModel.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs/promises");
+
+exports.selectEndpoints = () => {
+  return fs
+    .readFile(`/home/meeragopi/northcoders/projects/be-nc-news/endpoints.json`)
```

Comment:
This path (`/home/meeragopi`) wont work when you host this at the end of the week. I'd look at using `__dirname` instead.